src/main/io/serial.c

```diff
@@ -245,7 +245,7 @@ bool isSerialPortShared(const serialPortConfig_t *portConfig, uint16_t functionM
     return (portConfig) && (portConfig->functionMask & sharedWithFunction) && (portConfig->functionMask & functionMask);
 }
 
-serialPort_t *findSharedSerialPort(uint16_t functionMask, serialPortFunction_e sharedWithFunction)
+serialPort_t *findSharedSerialPort(uint32_t functionMask, serialPortFunction_e sharedWithFunction)
```

Comment:
I don't think this change should be part of this pull request.

Reply:
I had this in my local changes and accidentally added it to the commit. Amending it right now.